contracts/libraries/IporFusionMarkets.sol

```diff
@@ -48,4 +48,6 @@ library IporFusionMarkets {
     /// @dev Curve market
     uint256 public constant CURVE_POOL = 16;
     uint256 public constant CURVE_LP_GAUGE = 17;
+
+    uint256 public constant RAMSES_V2_POSITIONS = 18;
```

Comment:
The error that fixes this was caused by the randomness of the data I entered, but I preferred to secure it. The original library (Uniswap) doesn't have this safeguard. However, it might be useful to deploy a new version.